components/script/dom/fontface.rs

```diff
@@ -540,6 +543,14 @@ impl FontFaceMethods<crate::DomTypeHolder> for FontFace {
         )
         .expect("Parsing shouldn't fail as descriptors are valid by construction");
 
+        //Construct a WebFontDocumentContext object for the current document.
+        let document_context = WebFontDocumentContext {
+            policy_container: global.policy_container(),
+            document_url: global.api_base_url(),
+            has_trustworthy_ancestor_origin: global.has_trustworthy_ancestor_origin(),
+            insecure_requests_policy: global.insecure_requests_policy(),
+        };
```

Comment:
Since this struct is only applicable on `Window` currently, perhaps it would be useful to create a helper in `Window` which makes this data structure.